docker-entrypoint.sh

```diff
@@ -11,6 +11,16 @@ find /src -mindepth 1 -maxdepth 1 \( -type d -name ".*" -prune \) -o -exec cp -r
 # Tox will be run by the "tox" user, so it should own /app/.
 find /app -name .tox -prune -o -print0 | xargs -0 chown tox:tox
 
+DOCKER_SOCKET=/var/run/docker.sock
+DOCKER_GROUP=docker
+REGULAR_USER=tox
+
+if [ -S ${DOCKER_SOCKET} ]; then
```

Comment:
Can we update this to be `if [ -S "${DOCKER_SOCKET}" ]; then` just to be safe

Reply:
if it was user input i'd agree, but it's not, it's hardcoded above and paths shouldn't have spaces in them, so i'm OK as is :man_shrugging: